independent-projects/parent/pom.xml

```diff
@@ -48,8 +48,6 @@
 
         <!-- Default properties -->
         <maven.compiler.parameters>true</maven.compiler.parameters>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
Could you please drop this commit from there? It has nothing to do with the issue at hand and I'm not sure it's worth taking any risk.